### PR TITLE
Remove electron references

### LINKED ISSUE #170: Update README and remove all documentation references to electron / UI

### PR DESCRIPTION
**Pull request by ejfox**
**Summary**

Removes references to electron and makes it so that the script can be run by calling `dataproofer file.csv` when installed using `npm install -g` 
Screen Shot 2021-03-03 at 2 08 01 PM

- Closes #170

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,18 @@
 {
+  "name": "dataproofer",
```
Comment (newsroomdev) [inline package.json:2]: interesting! silly question, won't this cause a name conflict with the other package.json named "dataproofer"? https://github.com/dataproofer/Dataproofer/blob/dev/src/package.json#L2

File: package.json [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
+  "description": "A proofreader for your data",
+
   "private": true,
   "workspaces": [
     "src",
     "packages/*"
   ],
+  "main": "./src/index.js",
+  "bin": {
+    "dataproofer": "./src/index.js"
```
Comment (newsroomdev) [inline package.json:13]: oh smart!

**Review by newsroomdev — COMMENTED** [review]
one question about name-spacing but this is looking good!

File: package.json (hunk 1 above)
Comment (newsroomdev) [reply package.json:2]: oh also seeing all the tests are failing now hm

File: package.json (hunk 2 above)
Comment (newsroomdev) [reply package.json:13]: i think you may need to update some of the workflows to get this passing

File: package.json (hunk 1 above)
Comment (ejfox) [reply package.json:2]: I think it's likely this is that's what is breaking it, maybe it should be `dataproofer-cli`

**Comment by ejfox** [comment]
Should pull in latest yarn version updates

**Review by newsroomdev — APPROVED** [review]

**Pull request merged**